Makefile

```diff
@@ -2,7 +2,7 @@ dev:
 	rm -rf ./server/dist && docker-compose -f ./docker/docker-compose.dev.yml up --remove-orphans
 
 dev-new:
-	rm -rf ./server/dist && docker compose -f ./docker/docker-compose.dev.yml up --remove-orphans
+	rm -rf ./server/dist && docker-compose -f ./docker/docker-compose.dev.yml up --remove-orphans
```

Comment:
You can use `make dev` if you need `docker-compose`. `dev-new` is for `docker compose`

Reply:
I believe `docker-compose` is aliased to `docker compose` on systems that use the latter, so we -should?- be safe to use the dash version everywhere.